recipe/conda_build_config.yaml

```diff
@@ -5,5 +5,5 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
 libevent:
   - "2.1.10"
   - "2.1.12"
-msgpack_c:
-  - 6
+lmdb:
+  - 0.9.22
```

Comment:
Note to reviewers: this is an alternative specification to make ArcticDB use LMDB 0.9.22.

Newer versions of LMDB could be used but they cause 1 parametrized tests out of 5000 to fail upstream. See: https://github.com/man-group/ArcticDB/pull/437